packages/devtools_app_shared/lib/src/ui/theme/ide_theme.dart

```diff
@@ -16,12 +16,14 @@ final class IdeTheme {
     this.foregroundColor,
```

Comment:
are we even using the backgroundColor and foregroundColor in this class?

Reply:
Yep, it's used around here:

https://github.com/flutter/devtools/blob/66a49c3500ef867662638432c4afebbdd128ffda/packages/devtools_app_shared/lib/src/ui/theme/theme.dart#L40

It only gets set from the querystring if it's dart/light enough (since in VS Code we read it from the theme that we have no control of), and then in the code above we override the colour to that (so that it more closely matches your IDE theme).